### ISSUE TITLE
MCM Unresponsive

### ISSUE THREAD

**Issue by solson440** [psouza4/mediacentermaster#1892]
For a couple of months, now, MCM becomes unresponsive. When this happens, the MCM window cannot be moved, resized or minimized. The only way to recover MCM is to stop viz Task Manager and start it back up. It may happen again shortly, or hours later. Task Manager does not show issues (unresponsive, etc) and is able to stop it with no issues. Using V2.24 build 13622.224. There are others having this issue on the MCM forum, as well. Seems to happen when scanning media folders.

**Comment by WhiskeySheppard** [comment]
no response to this here either I see ;-).

**Comment by BigChaps** [comment]
I have had this happeneing a lot too on multiple machines that I have tested on. Had worked bpreviously with large library.

- Plex Server has 1634 Titles across local HDDs and USB3 external HDDs 
- My Main PC scans 3792 Titles across local HDDs, USB3 external HDDs and SMB mapped network drives

Works for a bit but random timing out, usually when scanning or updating the metadata. Often the main title window freezes first while the log scrolls but no always. Normal exit often doesn't close properly, freezing on writing cache.

**Version Info - Plex Server**
Media Center Master: 2.24.13622.224
Windows version: Windows 10 Pro (64-bit)
Windows version (raw): Microsoft Windows NT 6.2.9200.0
.NET framework information:
	Runtime loaded: 2.0.50727.9164
	Runtimes installed:
		2.0.50727.4927 SP2
		3.0.30729.4926 SP2
		3.5.30729.4926 SP1
		4.8.04084
		4.0.0.0

**Version info - Main PC**
Media Center Master: 2.22.33621.1071
Windows version: Windows 10 Pro (64-bit)
Windows version (raw): Microsoft Windows NT 6.2.9200.0
.NET framework information:
	Runtime loaded: 2.0.50727.9164
	Runtimes installed:
		2.0.50727.4927 SP2
		3.0.30729.4926 SP2
		3.5.30729.4926 SP1
		4.8.04084
		4.0.0.0

**Comment by madmax2** [comment]
same here, the program is laggy UI and often becomes unresponsive after finishing a scan.
Then need to force close/kill the program in task manager then restart again.

**Comment by madmax2** [comment]
MCM forum post about same issue

http://forums.mediacentermaster.com/viewtopic.php?f=6&t=20624&p=63197#p63197

**Comment by austwhite** [comment]
Just messaging here as I have same issues, but for me it is much logner than a couple of months .  Freezing, becoming unresponsive, needing to force close has been part of MCM for years.  
To be frank, it is bad enough that if I had an alternative that did all it did, I wouldn't even bother with MCM anymore.   Especially bad if there is a larger library involved.   
It has done it on Windows 7, Windows 8, Windows 10 and Windows 11.
Even on shutdown MCM often freezes with a screen saying it is writing to a cache file.    
It does it less when run as admin, but still does it consistantly.

I am using a 12th Gen Core i5, 16GB DDR5 RAM and a 512GB SSD for the database

**Comment by hounddog59** [comment]
Dito too all of the above comments I paid for premium a long time ago.
Its time to hand the raines to Doug I think, give him the code so we can have a solid program again. Thank you for what you have done Peter. but if you are unable then let someone else.

**Comment by madmax2** [comment]
@hounddog59 not sure if Doug is a coder  

seems like he is mainly helping with support in the forum

**Comment by hounddog59** [comment]
Yeah but is he really? there's nothing been posted for a while and what has been ... no response are talking discord?
cheers

**Comment by madmax2** [comment]
@hounddog59 the program is very much dead in terms of wanting to improve or add features..

The program has no benefit to anyone who still thinks it is worthwhile paying for it anymore
due to the poor performance, laggy UI and responsive and not fixing it at all.
No new suggest features have been implemented either.

Don't know how many people still paying for premium.

Would be good if the owner @psouza4 made the code open source for other people who can help fix this issue or take over.
Cos he seems to have no interest in improving this program anymore

**Comment by hounddog59** [comment]
totally agree just logged into discord to see it in there and mmmm not sure
